<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="name_lb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>이름 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_lb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Email :</string>
   </property>
  </widget>
  <widget class="QLabel" name="pw_lb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PassWord :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_email">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_password">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
